ts/features/bonusVacanze/components/loadingErrorScreen/LoadingErrorComponent.tsx

```diff
@@ -1,18 +1,52 @@
+import { Millisecond } from "italia-ts-commons/lib/units";
+import { Spinner, View } from "native-base";
 import * as React from "react";
+import { useEffect } from "react";
 import { SafeAreaView } from "react-native";
-import { withLoadingSpinner } from "../../../../components/helpers/withLoadingSpinner";
 import GenericErrorComponent from "../../../../components/screens/GenericErrorComponent";
+import { setAccessibilityFocus } from "../../../../utils/accessibility";
+import { InfoScreenComponent } from "../infoScreen/InfoScreenComponent";
 import { bonusVacanzeStyle } from "../Styles";
 
 export type LoadingErrorProps = {
   isLoading: boolean;
-  loadingCaption?: string;
-  loadingOpacity?: number;
+  loadingCaption: string;
   errorText?: string;
   onRetry: () => void;
   onAbort?: () => void;
 };
 
+const errorRef = React.createRef<GenericErrorComponent>();
+const loadingRef = React.createRef<View>();
+
+const renderError = (props: LoadingErrorProps) => (
+  <GenericErrorComponent
+    onRetry={props.onRetry}
+    onCancel={props.onAbort}
+    text={props.errorText}
+    subText={""}
+    ref={errorRef}
+  />
+);
+
+const renderLoading = (loadingCaption: string) => (
+  <View accessible={true} ref={loadingRef} style={{ flex: 1 }}>
+    <InfoScreenComponent
+      image={
+        <Spinner
```

Comment:
```suggestion
        <ActivityIndicator color={themeVariables.brandPrimaryLight} />
```
you could use ActivityIndicator since it is already used in few parts (`/components/ui/ActivityIndicator`)

| Spinner black  | ActivityIndicator |
| ------------- | ------------- |
![Spinner black](https://user-images.githubusercontent.com/822471/87406636-424e2900-c5c1-11ea-8ec7-a30e311de94d.jpg) | ![ActivityIndicator](https://user-images.githubusercontent.com/822471/87406431-0024e780-c5c1-11ea-858a-ed8f7aba9074.jpg)


Furthermore we could keep the InfoScreenComponent  title as non-bold text as it was previously
### current on master
<img src="https://user-images.githubusercontent.com/822471/87406542-2054a680-c5c1-11ea-996e-4efb153de81a.jpg" height="600"/>

Reply:
LoadingSpinner changed in https://github.com/pagopa/io-app/pull/2043/commits/e677b097067713a2cc7c6b1a1b3909f3f99982bb

The text is changed to bold in order to use this pr as an occasion to implements the graphical requirement that was never implemented https://pagopa.invisionapp.com/console/Bonus-Vacanze-ckas11mrt0cfe016bekk47kk5/ckb19iapa0036013eyble6agi/play

Just the subtext cannot be `normal` for accessibility reasons.